.travis.yml

```diff
@@ -15,7 +15,7 @@ env:
 before_install:
     # Fast finish the PR.
     - |
-      (cat recipe/ff_ci_pr_build.py | \
+      (curl https://raw.githubusercontent.com/conda-forge/conda-forge-build-setup-feedstock/master/recipe/ff_ci_pr_build.py | \
```

Comment:
This should still be ok. It's just using the file already in `recipe` independently of installing the package.